### DISCUSSION TITLE
Is there a way to get rid of zebra striping?

### DISCUSSION THREAD

**Discussion by dambridge** [rappasoft/laravel-livewire-tables#851] [General]
Still digging, but haven't figured a way to have all rows the same background.

**Comment by dgillier** [comment]
You can use ->setTrAttributes() to change the tr class.
(https://rappasoft.com/docs/laravel-livewire-tables/v2/datatable/available-methods)

You can set the default param to true or false to replace classes or just add a new one to.

```
public function configure(): void
{
  // Takes a callback that gives you the current row and its index
  $this->setTrAttributes(function($row, $index) {
        return [
           'default' => true,
          'class' => 'bg-gray-200',
        ];
  });
}
```

**Comment by hasan-ozbey** [comment]
Tables have `table table-striped` classes by default. You need to override class list and remove the `table-striped` class.

```php
public function configure(): void
{
  // ...

  $this->setTableAttributes([
    'default' => false,
    'class' => 'table',
  ]);
}
```
